backends/advanced/notebooks/debugmemory.ipynb

```
@@ -0,0 +1,204 @@
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0, \"..\")\n",
+    "sys.path.insert(1, \"src\")\n",
+    "import os\n",
+    "\n",
+    "# Set environment variables\n",
+    "os.environ[\"OPENAI_API_KEY\"] = \"ollama\"  \n",
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)